maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -412,3 +412,27 @@ data class RunFlowCommand(
     }
 
 }
+
+data class SetLocationCommand(
+    val latitude: Double,
+    val longitude: Double,
+) : Command {
+
+    override fun description(): String {
+        return "Set location (${latitude}, ${longitude})"
+    }
+
+    override fun injectEnv(env: Map<String, String>): SetLocationCommand {
+        return this
+    }
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
`equals` and `hashCode` are not needed for data classes, we only use it for commands that don't have any fields

Reply:
Done